### PR TITLE
Update manage-sites-in-new-admin-center.md

### PR DESCRIPTION
**Pull request by PramodBalusu**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ The Active sites page of the new SharePoint admin center lets you view the Share
 ![Manage sites in the new SharePoint admin center](media/2a18e27e-47ba-4370-8d91-cb6d75d746b5.png)
   
 > [!NOTE]
-> The Active sites page lists the root website for each site collection. Subsites aren't included in the list. <br>Some functionality is introduced gradually to organizations that have opted in to the [Targeted release option in Microsoft 365](/office365/admin/manage/release-options-in-office-365). This means that you might not yet see some features described in this article, or they might look different. 
+> The Active sites page lists the root website for each site collection. Subsites, REDIRECTSITE#0 (used for Renamed/swapped sites redirect) and TEAMCHANNEL#0(Private team channel sites) template sites aren't included in the list. <br>Some functionality is introduced gradually to organizations that have opted in to the [Targeted release option in Microsoft 365](/office365/admin/manage/release-options-in-office-365). This means that you might not yet see some features described in this article, or they might look different.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md:32]:
```suggestion
> The Active sites page lists the root website for each site collection. Subsites, REDIRECTSITE#0 (used for Renamed/swapped sites redirect), and TEAMCHANNEL#0 (Private team channel sites) template sites aren't included in the list. <br>Some functionality is introduced gradually to organizations that have opted in to the [Targeted release option in Microsoft 365](/office365/admin/manage/release-options-in-office-365). This means that you might not yet see some features described in this article, or they might look different. 
```

Comment (JohanFreelancer9) [reply SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md:32]: Please insert a comma and space as indicated. "... sites redirect**), a**nd TEAMCHANNEL#**0 (Pr**ivate ...".

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**